layouts/partials/scripts.html

```diff
@@ -5,3 +5,10 @@
 <script type="text/javascript" src="{{ "/js/navbar.js" | relURL }}"></script>
 <script type="text/javascript" src="{{ "/js/plyr.js" | relURL }}"></script>
 <script type="text/javascript" src="{{ "/js/main.js" | relURL }}"></script>
+
+{{ if site.Params.darkTheme.enable }}
+{{ if eq site.Params.darkTheme.provider "darkreader" }}
+<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/darkreader@4.9.51/darkreader.js"></script>
```

Comment:
Let's use local file instead of CDN. Some people seems to be concerned with CDN. Include the file in js folder.

Reply:
Added a modified copy of darkreader.js.